src/Result/Graphql/Order/OrderCreateResultHandler.php

```diff
@@ -57,7 +56,5 @@ protected function parseQueryResult(\stdClass $stdClass)
                 return $this->parseQueryResult($member);
```

Comment:
Let's check the response code here. The success / failure of an API call should not be detected based on the response body